spk/python310/src/requirements-abi3.txt

```diff
@@ -8,5 +8,5 @@
 # Require:
 #    - CFLAGS=-sdt=c99 <  gcc-4.9
 #    - CFLAGS=-sdt=c11 >= gcc-4.9
```

Comment:
typo:  std (not sdt)

Reply:
Good catch... Gladly its ok in the makefile :)